table_foosball/database_test.rb

```diff
@@ -0,0 +1,79 @@
+require_relative './database'
+require_relative './table_foosball_01'
+require 'minitest/autorun'
+
+class DatabaseTest < Minitest::Test
+	def setup
```

Comment:
I think you should decouple these tests from Games and Players. Instead, define some custom classes directly in here and use them, e.g.

```ruby
class DatabaseTest < Minitest::Test
  class Foo
    def initialize(..)
    end
  end
  
  def setup
    # stuff
  end

  # tests
end